js/library.js

```diff
@@ -1,16 +1,140 @@
-function Book(title, pages, status) {
-    this.title = title
-    this.pages = pages
-    this.status = status
-    this.info = function() {
-        let text = 'already read'
-        let result = ''
-        if (status = true) {
-            text = 'not read yet'
-        }
-        return title.concat(title + pages + 'pages' + text)
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+/* global firebase */
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyAkEY49W_RKvLI6rcO_r6SjnrJUvwduu2U',
```

Comment:
## Modularise this module 
- [ ] Break this `library` module into separate files then import the functions in a file you would want to be the entry point to the application. Let's make sure we keep the application logic separate from DOM manipulation tasks ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright)